tests/runner.py

```diff
@@ -272,7 +273,7 @@ def run_tests(options, suites):
   print('Test suites:')
   print([s[0] for s in suites])
   # Run the discovered tests
-  testRunner = unittest.TextTestRunner(verbosity=2)
+  testRunner = unittest.TextTestRunner(verbosity=2, failfast=options.failfast)
```

Comment:
How about a separate PR for this addition?

Reply:
ok